src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
Here for ref: https://github.com/react-community/react-navigation/pull/109/commits/1b5fb64cd376723ba0e3c044322595741e2009f8